From bb99da75b1f1ec198015a2b43dcb42bfa42085be Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Thu, 11 Apr 2019 14:01:06 +0100
Subject: [PATCH] TOOLS-2240 smartos-release script needs filename update for
 loader images Reviewed by: John Levon <john.levon@joyent.com> Approved by:
 John Levon <john.levon@joyent.com>

---
 tools/smartos-release | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tools/smartos-release b/tools/smartos-release
index 9dd343d..c9b270f 100755
--- a/tools/smartos-release
+++ b/tools/smartos-release
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright 2019, Joyent, Inc.
 #
 
 # Update the SmartOS release directory
@@ -77,8 +77,8 @@ done
 echo "Updating top level ${SMARTOS} snaplinks"
 mln ${SOURCE}/platform-${TIMESTAMP}.tgz ${SMARTOS}/platform-latest.tgz
 mln ${SOURCE}/smartos-${TIMESTAMP}.iso ${SMARTOS}/smartos-latest.iso
-mln ${SOURCE}/smartos-${TIMESTAMP}-USB.img.bz2 ${SMARTOS}/smartos-latest-USB.img.bz2
-mln ${SOURCE}/smartos-${TIMESTAMP}.vmwarevm.tar.bz2 ${SMARTOS}/smartos-latest.vmwarevm.tar.bz2
+mln ${SOURCE}/smartos-${TIMESTAMP}-USB.img.gz ${SMARTOS}/smartos-latest-USB.img.gz
+mln ${SOURCE}/smartos-${TIMESTAMP}.vmwarevm.tar.gz ${SMARTOS}/smartos-latest.vmwarevm.tar.gz
 
 echo "Updating ${SMARTOS}/latest object"
 echo ${DESTINATION} | mput -v -H 'content-type: text/plain' ${SMARTOS}/latest
-- 
2.21.0

